packages/vscode-extension/src/plugins/network/strategies/BaseNetworkInspector.ts

```diff
@@ -0,0 +1,160 @@
+import { Disposable } from "vscode";
+import { NetworkInspector, BroadcastListener } from "../network-plugin";
+import {
+  WebviewMessage,
+  IDEMessage,
+  WebviewCommand,
+  IDEMethod,
+} from "../../../network/types/panelMessageProtocol";
+import { RequestData, RequestOptions } from "../../../network/types/network";
+import { Logger } from "../../../Logger";
+import { determineLanguage } from "../../../network/utils/requestFormatters";
+import { openContentInEditor, showDismissableError } from "../../../utilities/editorOpeners";
+import { extractTheme } from "../../../utilities/themeExtractor";
+import { ContentTypeHeader } from "../../../network/types/network";
+
+export abstract class BaseInspectorStrategy implements NetworkInspector {
```

Comment:
```suggestion
export abstract class BaseNetworkInspector implements NetworkInspector {
```
you probably used vsocde's rename in the wrong place, and renamed an import instead of this class.